books/RayTracingTheNextWeek.html

```diff
@@ -392,10 +392,14 @@
         auto dist_to_focus = 10.0;
         auto aperture = 0.1;
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
+        const auto time_start = 0.0;
+        const auto time_end = 1.0;
```

Comment:
This belongs in a separate issue, but we should consider moving `aperture`, `dist_to_focus`, `time_start`, `time_end` to above the scene switch. After being introduced, we never really change these values, but _someone_ probably does

Reply:
Good idea. I'll add that.

Reply:
Done.